saleor/payment/gateways/stripe/plugin.py

```diff
@@ -328,6 +354,31 @@ def void_payment(
             raw_response=raw_response,
         )
 
+    @require_active_plugin
+    def list_payment_sources(
+        self, customer_id: str, previous_value
+    ) -> List[CustomerSource]:
+        payment_methods, error = list_customer_payment_methods(
+            api_key=self.config.connection_params["secret_api_key"],
+            customer_id=customer_id,
+        )
+        if payment_methods:
+            customer_sources = [
+                CustomerSource(
+                    id=c.id,
+                    gateway="stripe",
```

Comment:
Shouldn't we have some enum here?

Reply:
Good catch, we should use here plugin_ID. Thanks @maarcingebala  ❤️